internal/pkg/config/model.go

```diff
@@ -45,8 +45,10 @@ const (
 	// The following enables OpenCensus Tracing via Cloud Trace
 	// for the Datastore client library.
 	// It is EXPERIMENTAL and subject to change or removal without notice.
-	EnableTrace     = "enable_trace"
```

Comment:
Can you please change the comments here to mention that the new integration is using Open Telemetry in place of OpenCensus?

Reply:
Done, I forgot about those, thanks